widgetastic_manageiq.py

```diff
@@ -878,3 +873,52 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class AlertEmail(View):
+    """Represents email notifications form when you are creating an alert."""
+
+    ROOT = ParametrizedLocator("//div[@id='edit_email_div']")
```

Comment:
Maybe you could smuggle the edit_email_div as a value for the actual parametrization? To give us a bit of flexibility?